app/assets/stylesheets/notice.scss

```diff
@@ -0,0 +1,17 @@
+.notice {
+  background-color: #D2D6DB; // smoke-20
```

Comment:
As mentioned in other comment, lets try to keep these as variables but not in global file since we're stepping away from using global file with variable.

Reply:
I've removed this file completely but now I'm remember to make lower-case hex codes wherever it isn't — glad to hear about this standard!